digital_image_processing/change_contrast.py

```diff
@@ -0,0 +1,35 @@
+"""
+Changing contrast with PIL
+
+This algorithm is used in
+https://noivce.pythonanywhere.com/ python web app.
+
+python/black: True
+flake8 : True
+"""
+
+from PIL import Image
+
+
+def change_contrast(img: Image, level: int) -> Image:
+    """
+    Function to change contrast
+    """
+    factor = (259 * (level + 255)) / (255 * (259 - level))
+
+    def contrast(c: int) -> float:
+        """
+        Fundamental Transformation/Operation that'll be performed on
+        every bit.
+        """
+        return 128 + factor * (c - 128)
+
+    return img.point(contrast)
+
+
+if __name__ == "__main__":
+    # Load image
+    img = Image.open("image_data/lena.jpg")

```

Comment:
A [slightly better approach](https://pillow.readthedocs.io/en/stable/reference/open_files.html#image-lifecycle) would be: __with Image.open() as img:__ and then indent the remaining lines.  That would close the image file when the _with_ block ends.  Leaving files open is not a good idea especially image files which can take up a lot of memory.

Reply:
Done